app/controllers/audits/base_controller.rb

```diff
@@ -7,6 +7,7 @@ def build_filter
       Filter.new(
         allocated_to: params[:allocated_to],
         audit_status: params[:audit_status],
+        current_user_id: current_user.uid,
```

Comment:
I feel like the concern of how to build the filter belongs in the controller. The controller can decide that if the `allocated_to` param is missing then we should apply a default.

Something like this:

```rb
Filter.new(allocated_to: params.fetch(:allocated_to, current_user.uid))
```